.werft/build.ts

```diff
@@ -547,7 +547,7 @@ export async function deployToDevWithInstaller(deploymentConfig: DeploymentConfi
     try {
         exec(`kubectl delete -n ${deploymentConfig.namespace} job migrations || true`,{ slice: installerSlices.APPLY_INSTALL_MANIFESTS, silent: true });
         // errors could result in outputing a secret to the werft log when kubernetes patches existing objects...
-        exec(`kubectl apply -f k8s.yaml`,{ slice: installerSlices.APPLY_INSTALL_MANIFESTS, silent: true });
+        exec(`kubectl apply -f k8s.yaml --validate=false`,{ slice: installerSlices.APPLY_INSTALL_MANIFESTS, silent: true });
```

Comment:
Why disable validation? Is our Kubernetes version too old, or does this change introduce backwards incompatible effects?

Reply:
I see this error without validate=false

```
Installing preview environment.
Error from server (NotFound): jobs.batch "migrations" not found

Warning: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
Warning: policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+, unavailable in v1.25+; use policy/v1 PodDisruptionBudget
error: error validating "k8s.yaml": error validating data: ValidationError(StatefulSet.status): unknown field "availableReplicas" in io.k8s.api.apps.v1.StatefulSetStatus; if you choose to ignore these errors, turn validation off with --validate=false
```

Reply:
> Is our Kubernetes version too old, or does this change introduce backwards incompatible effects?

We need an update in the core-dev cluster

Reply:
We currently require Kubernetes 1.21 for self-hosted installations. I wonder what this change will do in the self-hosted case? 

I.e. will others be facing the same validation errors, or is this only a problem because we run `kubectl 1.22` against a `1.21` cluster?